android/pytorch_android/src/androidTest/java/org/pytorch/PytorchHostTests.java

```diff
@@ -1,28 +1,22 @@
 package org.pytorch;
 
-import com.facebook.soloader.nativeloader.NativeLoader;
```

Comment:
It looks like this file was reformatted.  Can you do that as a separate diff first to make this change more self-contained?